<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_altadocumento</class>
 <widget class="QDialog" name="form_altadocumento">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Formulario alta de archivo</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(82, 175, 50);
font: 75 12pt &quot;Courier 10 Pitch&quot;;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(29, 67, 111);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="2" column="0">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QTabWidget" name="tabWidget">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(211, 215, 207);</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Guardar archivo</string>
         </attribute>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>140</y>
            <width>100</width>
            <height>19</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>ID:</string>
          </property>
         </widget>
         <widget class="QLabel" name="lbl_id">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>140</y>
            <width>61</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>asd</string>
          </property>
         </widget>
         <widget class="QPushButton" name="boton_buscar">
          <property name="geometry">
           <rect>
            <x>630</x>
            <y>40</y>
            <width>80</width>
            <height>27</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(82, 175, 50);</string>
          </property>
          <property name="text">
           <string>buscar</string>
          </property>
         </widget>
         <widget class="QLabel" name="lbl_buscar">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>40</y>
            <width>511</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>asd</string>
          </property>
         </widget>
         <widget class="QPushButton" name="boton_guardar">
          <property name="geometry">
           <rect>
            <x>270</x>
            <y>140</y>
            <width>80</width>
            <height>27</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(82, 175, 50);</string>
          </property>
          <property name="text">
           <string>Guardar</string>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>11</y>
            <width>154</width>
            <height>25</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>25</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Usuario :  Admin</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>87</width>
            <height>25</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>25</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>archivo: </string>
          </property>
         </widget>
         <zorder>label_3</zorder>
         <zorder>boton_buscar</zorder>
         <zorder>lbl_id</zorder>
         <zorder>lbl_buscar</zorder>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
